api/custom/controllers/custom.js

```diff
@@ -10,6 +10,13 @@ module.exports = {
         maxAge: 1000 * 60 * 60 * 24 * 14, // 14 Day Age,
         domain: 'localhost'
       });
+    } else if(process.env.NODE_ENV === 'staging') {
```

Comment:
Instead of hard coding the domain, we can make it configurable and simplify `if, else if, else` to `if else`.

Reply:
Oh you mean by making domain into an environment variable and setting it to staging.devlaunchers.com in staging?

Reply:
Yes.